backend/apps/residents/services.py

```
@@ -58,6 +59,77 @@ def _get_student_role() -> Role:
+    password: str | None,
+    request,
+) -> None:
+    if password:
+        user.set_password(password)
+        user.save()
+        return
+
+    try:
+        process_password_reset_request(user.email, request)
+    except Exception:
+        pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)